.github/workflows/publish_npm.yml

```
@@ -0,0 +1,25 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
```

Why an old version of node? Just use the latest LTS
```suggestion
          node-version: 'lts/*'
          cache: yarn
```